<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="TokenTextEdit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>90</y>
      <width>390</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="headerTextEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="PayloadTextEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="SIgnatureTextEdit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="HeaderLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>391</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>HEADER: ALGORITHM &amp; TOKEN TYPE</string>
    </property>
   </widget>
   <widget class="QLabel" name="PayloadLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PAYLOAD: DATA</string>
    </property>
   </widget>
   <widget class="QLabel" name="SignatureLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>391</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VERIFY SIGNATURE</string>
    </property>
   </widget>
   <widget class="QLabel" name="EncodedLabel">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>40</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoded</string>
    </property>
   </widget>
   <widget class="QLabel" name="DecodedLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>100</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>Decoded</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
